migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql

```diff
@@ -0,0 +1,138 @@
+-- Show_spuSOLRLineOfServiceGenerateFromMid
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRLINEOFSERVICE() 
```

Comment:
Add the Database to the procesure name
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRLINEOFSERVICE() 
```

Reply:
Done